linkis-computation-governance/linkis-manager/linkis-manager-persistence/src/main/java/org/apache/linkis/manager/dao/LockManagerMapper.java

```diff
@@ -25,21 +25,12 @@
 
 @Mapper
 public interface LockManagerMapper {
-  @Insert(
-      "insert into linkis_cg_manager_lock (lock_object, time_out, update_time, create_time)"
-          + "values(#{jsonObject}, #{timeOut}, now(), now())")
+
   void lock(@Param("jsonObject") String jsonObject, @Param("timeOut") Long timeOut);
 
-  @Delete("delete  from linkis_cg_manager_lock where id = #{id}")
   void unlock(Integer id);
 
-  @Select("select * from  linkis_cg_manager_lock where lock_object = #{jsonObject}")
   List<PersistenceLock> getLockersByLockObject(String jsonObject);
```

Comment:
For consistency, please add parameter annotations 
`@param("jsonObject") String jsonObject ->@param("lockObject") String lockObject`